bin/test-example

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+
+if [ $# -eq 0 ]; then
+    echo "usage: $0 /path/to/exercise/example"
+    exit 1
+fi
+
+exampledir=$(cd "$1" && pwd)
+
+examplename=$(basename "$exampledir")
+exercisedir=$(cd "$exampledir/../.." && pwd)
+exercisename=$(basename "$exercisedir")
+xhaskell="$(cd "$(dirname "$0")/.." && pwd)"
+
+buildfolder=$(mktemp -d)
```

Comment:
> the build path needs to be a function from (exercise, example/stub).

Oh wow, so I can't even place it in the current directory (like I planned to).

I'll try always putting it in xhaskell then.